.github/workflows/pull_request_checks.yml

```diff
@@ -0,0 +1,57 @@
+name: Pull Request Checks
+
+on:
+  pull_request:
+    types:
+      - opened
+      - edited
+      - synchronize
+
+jobs:
+  name_check:
+    name: Pull Request Name Check
+    runs-on: ubuntu-latest
+    env: 
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - name: Clone Repository
+        uses: actions/checkout@v2
+
+      - name: Install Node
+        uses: actions/setup-node@v2.2.0
+        with:
+          node-version: 16
+          cache: npm
+      
+      - name: Install NPM Dependencies
+        run: npm ci
+
+      - name: Ensure Commitizen Format
+        uses: JulienKode/pull-request-name-linter-action@98794a8b815ec05560813c42e55fd8d32d3fd248
+
+  size_label:
+    name: Change Size Label
```

Comment:
Another thing I like but we can totally delete it. Labels the PR based on the size of the legitimate code changes (ignores `package-lock.json` changes, for instance).